src/main/kotlin/com/nextstep/blackjack/controller/Main.kt

```diff
@@ -11,23 +12,14 @@ fun main() {
     OutputView.printStartMessage()
 
     val playersInput = InputView.inputMessageSplitWithComma()
-    val players = playersInput.map { Player(it) }
+    val players = Players(playersInput.map { Player(it) })
```

Comment:
이러한 구문을 함수 reference로 넘길 수 있습니다. 
```suggestion
val players = Players(playersInput.map(::Player))
```

lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 🙂 
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845